buildSrc/src/main/java/Dependencies.kt

```diff
@@ -65,79 +65,109 @@ object Deps {
 
     const val mozilla_concept_engine = "org.mozilla.components:concept-engine:${Versions.mozilla_android_components}"
     const val mozilla_concept_push = "org.mozilla.components:concept-push:${Versions.mozilla_android_components}"
-    const val mozilla_concept_tabstray = "org.mozilla.components:concept-tabstray:${Versions.mozilla_android_components}"
```

Comment:
Can you  revert the formatting changes to all these lines? That way our git blame won't pick up the wrong changes.

Reply:
Will do